subsys/net/ip/net_tc_mapping.h

```diff
@@ -0,0 +1,140 @@
+/** @file
+ * @brief Various priority to traffic class mappings
+ *
+ * This is not to be included by the application.
+ */
+
+/*
+ * Copyright (c) 2018 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __TC_MAPPING_H
+#define __TC_MAPPING_H
+
+#include <zephyr/types.h>
+
+/* All the maps below use priorities and indexes, below is the list of them
+ * according to 802.1Q - table I-2.
+ *
+ *  Priority         Acronym   Traffic types
+ *  1 (lowest)       BK        Background
```

Comment:
Please fix also the NET_PRIORITY_xx settings in net_ip.h

Reply:
Added a commit that does just that.